go/vt/schema/tablegc.go

```diff
@@ -44,85 +44,76 @@ const (
 	TableDroppedGCState TableGCState = ""
 )
 
+func (s TableGCState) TableHint() InternalTableHint {
+	switch s {
+	case HoldTableGCState:
+		return InternalTableGCHoldHint
+	case PurgeTableGCState:
+		return InternalTableGCPurgeHint
+	case EvacTableGCState:
+		return InternalTableGCEvacHint
+	case DropTableGCState:
+		return InternalTableGCDropHint
+	default:
+		return InternalTableUnknownHint
+	}
```

Comment:
Any reason not to make a map of these states like protobuf does? Not a big deal, but the use of Internal (capital I) and Hint here are not obvious to the average reader (me).